internal/server/storage/drivers/driver_zfs.go

```diff
@@ -273,9 +289,9 @@ func (d *zfs) Create() error {
 				return err
 			}
 		}
-	} else if filepath.IsAbs(d.config["source"]) {
+	} else if sliceAll(devices, func(device string) bool { return filepath.IsAbs(device) }) {
```

Comment:
Same issue as above, we need the `else if` and `else` statements flipped as described above to avoid incorrectly treating bad user input as an existing zpool name.

Reply:
Would the vdev type check at the start suffice?

Reply:
Unless I missed something, no, the check doesn't solve the problem.

Reply:
I've rearranged the blocks to align with feedback.